src/features/Main/sections/MainBrandingSection.tsx

```diff
@@ -0,0 +1,244 @@
+import React from 'react';
+import Image from 'next/image';
+import { css } from '@emotion/react';
+import { motion } from 'framer-motion';
+
+import { useCheckWindowSize } from '~/hooks/useCheckWindowSize';
+import { sectionBg } from '~/styles/background';
+import { colors } from '~/styles/colors';
+import { mediaQuery } from '~/styles/media';
+import { pxToRem } from '~/styles/style.utils';
+import { theme } from '~/styles/theme';
+
+/**
+ * MainBrandingSection : section 02
+ *
+ */
+export const MainBrandingSection = () => {
+  const { isTargetSize: isMobileSize } = useCheckWindowSize('mobile');
+
+  return (
+    <section css={layoutCss}>
+      <div css={wrapper}>
+        <div css={headerWrapper}>
+          <div css={titleWrapper}>
+            <div css={titleCss}>
+              {`Define\nyour direction`.split('').map((char, index) => (
+                <motion.span
+                  key={index}
+                  initial={{ opacity: 0 }}
+                  whileInView={{ opacity: 1 }}
+                  viewport={{ once: false, amount: 0.5 }}
+                  transition={{
+                    delay: 0.2 + index * 0.1,
+                    duration: 0,
+                  }}
+                >
+                  {char}
+                </motion.span>
+              ))}
+              {[...Array(4)].map((_, i) => (
+                <div id="circle" key={i}></div>
+              ))}
+            </div>
+          </div>
+          <p css={sentenceCss}>두려움을 용기로, 상상을 도전으로</p>
+        </div>
+        <div css={imagesWrapperCss}>
+          <Image
+            src={'/images/17th/3d-icon/web-icon.png'}
+            alt={'web-icon'}
+            width={!isMobileSize ? 300 : 200}
+            height={!isMobileSize ? 300 : 200}
+          />
+          <Image
+            src={'/images/17th/3d-icon/Android-icon.png'}
+            alt={'android-icon'}
+            width={!isMobileSize ? 300 : 200}
+            height={!isMobileSize ? 300 : 200}
+          />
+          <Image
+            src={'/images/17th/3d-icon/server-icon.png'}
+            alt={'server-icon'}
+            width={!isMobileSize ? 300 : 200}
+            height={!isMobileSize ? 300 : 200}
+          />
+        </div>
+      </div>
+    </section>
+  );
+};
+
+const layoutCss = css`
+  padding: 78px 0;
+  width: 100%;
+  height: auto;
+  display: flex;50% 0 32px 0;
```

Comment:
```suggestion
  display: flex;
```
display 속성 뒤에 뭔가가 붙어있어요..!

Reply:
어머 저걸 왜 못 봤지ㅜㅜ 알려주셔서 감사합니다!! 바로 수정할게요!